common/src/main/scala/uk/ac/wellcome/platform/archive/common/storage/services/SizeFinder.scala

```diff
@@ -28,13 +30,17 @@ class MemorySizeFinder(
 
 class S3SizeFinder(implicit s3Client: AmazonS3) extends SizeFinder {
   def getSize(location: ObjectLocation): Either[ReadError, Long] = {
-    val result =
-      S3Get
-        .get(location, maxRetries = 3) { location: ObjectLocation =>
-          s3Client
-            .getObjectMetadata(location.namespace, location.path)
-            .getContentLength
-        }
+
+    // TODO: Consider this being a "metadata readable"
+    // note this used to retry
```

Comment:
😬

Reply:
What do you reckon? Ensure we maintain retries here?

Reply:
If we can, yeah, otherwise the register could get flaky.